weather_mv/loader_pipeline/ee.py

```diff
@@ -459,15 +458,16 @@ def convert_to_asset(self, queue: Queue, uri: str):
                 attrs = ds.attrs
                 data = list(ds.values())
                 asset_name = get_ee_safe_name(uri)
-                channel_names = [da.name for da in data]
-                start_time, end_time, is_normalized = (attrs.get(key) for key in
-                                                       ('start_time', 'end_time', 'is_normalized'))
+                channel_names = list(self.band_names_dict.values()) \
+                    if self.band_names_dict \
+                    else [da.name for da in data]
+
                 dtype, crs, transform = (attrs.pop(key) for key in ['dtype', 'crs', 'transform'])
-                attrs.update({'is_normalized': str(is_normalized)})  # EE properties does not support bool.
```

Comment:
Is this removed?

Reply:
I removed it and there was no error produced while ingesting the asset to the Earth Engine. Should we add functionality for converting `bool` values to **string** in the `to_json_serializable_type` function?

Reply:
No need. This is fine. I was just making sure that we don't need this attribute.

Reply:
Marking as resolved